rfb/rfbclient.h

```diff
@@ -619,6 +619,9 @@ typedef struct _rfbClientProtocolExtension {
 	/** returns TRUE if it handled the message */
 	rfbBool (*handleMessage)(rfbClient* cl,
 		 rfbServerToClientMsg* message);
+	uint32_t const* securityTypes;
```

Comment:
Really, this should go to the end of the struct. As this may seem aesthetically unpleasing, here's why: Imagine a program built with libvncserver 0.9.11. The library will be updated to 0.9.12 at some point and let's say said program will not be rebuilt against the new lib. What happens is it will look for the  `next` pointer at a certain offset in the struct, but all of a sudden there will be `securityTypes` with different semantics. Boom.

Reply:
Ok, I wasn't aware of the fact that programs actually access/use the next pointer at all as it primarily is written/read by libvncclient internally. Anyway I'm fine with fixing this :)

Reply:
It's exposed as public API (unfortunately), so we cannot rule this out.

Reply:
No problem :) Anything left to be done for this?